modules/os2forms_nemid/src/Service/FormsHelper.php

```diff
@@ -20,6 +22,32 @@
  * @package Drupal\os2forms_nemid\Service
  */
 class FormsHelper {
+  const TEMPORARY_KEY = 'os2forms_nemlogin_address_protection';
```

Comment:
Why is it called “temporary key”? What's wrong with `ADDRESS_PROTECTION_STATE`, say?

Reply:
Renamed.